src/components/Todos.js

```diff
@@ -0,0 +1,131 @@
+import { setLocalStorage, store } from './storage';
+import { projectIdx, sanitizeName, removeChildDOM } from './Ui';
+
+const editForm = () => (`
+          <form>
+            <label>title
+              <input type="text" class="todo-title form-control edit-title"></input>
+            </label>
+            <label>
+              description
+              <input type="text" class="todo-title form-control edit-description"></input>
+            </label>
+            <label> Due Date
+              <input type="date" class="todo-title form-control edit-date"></input>
+            </label>
+            <select name="edit-priority" id="edit-priority" class="form-control">
+              <option value="high">High</option>
+              <option value="medium">medium</option>
+              <option value="low">low</option>
+          </select>
+            <button type="submit" class="edit-form-btn btn btn-primary">Edit</button>
+          </form>
+          `
+);
+
+// const completedTask = () => {
+//   const btns = document.querySelectorAll('.complete-btn');
+
+//   btns.forEach(btn => {
+//     btn.addEventListener('click', e => {
+//       const card = document.querySelector(`#${e.target.dataset.sucess}`);
+//       card.classList.toggle('border');
+//       card.classList.toggle('border-success');
+//     });
+//   });
+// };
```

Comment:
- [ ] Please remove this commented code block. Kindly use git history if you want to refer to previous code versions.